src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -33,7 +33,7 @@ export const DEFAULT_ROWS = 25;
 
 interface MapProps {
   user: FirebaseReducer.AuthState;
-  profileData: UserExperienceData;
+  profileData?: UserExperienceData;
```

Comment:
Looks like while this was a required prop here, the actually usages of it elsewhere in the code were treating it as though it could already be `undefined`.